### ISSUE TITLE
arm64 on Github Actions

### ISSUE THREAD

**Issue by richardburleigh** [cansik/openvino-arm#2]
Really nice work on this repo!

I've been trying to get Github Actions working [on my fork,](https://github.com/richardburleigh/openvino-arm) but no matter what cmake arguments I add, it always generates an x86 wheel.

How did you manage to generate arm builds?

**Comment by cansik** [comment]
I built the wheels on my developer device. Even it is possible to build arm64 binaries on intel architecture macs, it is difficult. What could possibly work is to change the `CC` variable to `CC='clang -arch=arm64'` before building, but I have not tried it yet.

**Comment by ilya-lavrenov** [comment]
You can currently try to build OpenVINO's 2022.3 branch with `-DCMAKE_OSX_ARCHITECTURES=arm64 .` and it should perform all magic

**Comment by cansik** [comment]
@ilya-lavrenov Thanks, I will give it a try. Is building an universal binary possible as well?

**Comment by ilya-lavrenov** [comment]
No, since ARM plug-in consists of NEON which can not be compiled for x86

**Comment by thusinh1969** [comment]
I compiled Openvino Arm successfully on Raspberry PI 4 and Orange PI and after a little sweating it ran OK, but the performance is terrible on Raspberry PI 4 (4G) vs i3/4G. On the new Orange PI 5 (8G) it is better but only 1/3 the performance of NUC i3 (Ubuntu) which is nearly the same price. ARM needs a lot of boosts to match Intel CPU. In the meanwhile, has anyone tried the NVIDIA plugin for Openvino? I tried with Jetson Nano without any luck keep asking for CUDART etc. 

Thanks for the pip repo anyway.
